main/swadge2024.c

```diff
@@ -407,6 +408,8 @@ void app_main(void)
 
     initLeds(GPIO_NUM_39, ledMirrorGpio, getLedBrightnessSetting());
 
+    initCh32v003(GPIO_NUM_18);
```

Comment:
I think you want `GPIO_SAO_2` here. That aliases to `GPIO_NUM_18` for the Wavebird & Gunship, but `GPIO_NUM_42` for the Hot Dog. See, configurable GPIO are good!

Reply:
Ok! Done.